packages/extension/src/providers/ethereum/networks/skale/nebula.ts

```diff
@@ -0,0 +1,25 @@
+import { NetworkNames } from "@enkryptcom/types";
+import {
+  SkaleParams,
+  createSkaleEvmNetwork,
+  ICustomSKALEAsset,
+} from "./skale-base";
+
+const skaleOptions: SkaleParams = {
+  name: NetworkNames.SkaleNebula,
+  name_long: "SKALE | Nebula Gaming Hub",
+  chainName: "green-giddy-denebola",
+  chainID: "0x585eb4b1",
+  icon: "skaleNebula.png",
+};
+
+const assets: ICustomSKALEAsset[] = [
+  {
+    name: "Ethereum Clone",
+    symbol: "ETHC",
+    address: "0xD2Aaa00700000000000000000000000000000000",
+    coingeckoID: "ethereum",
+  },
+];
```

Comment:
Remove, this token should never be used

Reply:
Removed - will add it later when we will switch to use the same token instead of wrapped